curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6465c4cb65beb801543de6e3.md

```diff
@@ -0,0 +1,706 @@
+---
+id: 6465c4cb65beb801543de6e3
+title: Step 83
+challengeType: 0
+dashedName: step-83
+---
+
+# --description--
+
+Now, you need to set the `href` attribute of the `a` tag to `${FORUM_TOPIC_URL}${slug}/${id}`. This will make the link point to the correct forum topic.
```

Comment:
```suggestion
Now, you need to set the `href` attribute of the `a` tag to `${FORUM_TOPIC_URL}${slug}/${id}`. This will make the link point to the correct forum topic.

With that last change, your freeCodeCamp forum leaderboard project is complete.
```